Make doesn't compile

I'm new to this disassembly project but when I run `make` it gives me an error:

Error: file not found (../../Zelda.gbc)Makefile:25: recipe for target 'bin/banks/bank_00_0.bin' failed

Is there more to this than simply running `make`?

--- a/Makefile
+++ b/Makefile
@@ -22,6 +22,9 @@ bin/banks:
 	mkdir -p bin/banks
 
 bin/banks/bank_00_0.bin: bin/banks DumpBanks
+ifeq (,$(wildcard Zelda.gbc))
+	$(error Original "Zelda.gbc" banks are missing. Find a copy of the "Legend of Zelda, The - Link's Awakening DX (U) (V1.0) [C][!]" (md5: 07C211479386825042EFB4AD31BB525F), and put it in the root directory to dump the banks)
+endif
 	cd bin/banks && ../../DumpBanks ../../Zelda.gbc
 
 # Objects are assembled from source.

--- a/README.md
+++ b/README.md
@@ -2,14 +2,17 @@
 
 Disassembly of one of my favorite games. Taking it easy for now.
 
-Thanks to these people for contributing:
-    * kemenaran - https://github.com/kemenaran
-    * Drenn1 - https://github.com/Drenn1
-
-Using rom
-
-```
-Legend of Zelda, The - Link's Awakening DX (U) (V1.0) [C][!].gbc
+Theses sources reproduce a valid build of `Legend of Zelda, The - Link's Awakening DX (U) (V1.0) [C][!].gbc`
 MD5 - 07C211479386825042EFB4AD31BB525F
-```
 
+## Usage
+
+1. Install [rbgds](https://github.com/rednex/rgbds#1-installing-rgbds).
+2. `make all`
+
+## Contributors
+
+Thanks to these people for contributing:
+
+* kemenaran - https://github.com/kemenaran
+* Drenn1 - https://github.com/Drenn1

